src/hand.rs

```diff
@@ -2,6 +2,7 @@ use crate::error::GamePlayError;
 
 use super::bag::TileBag;
 
+#[derive(Debug, PartialEq)]
 pub struct Hands {
     hands: Vec<Vec<char>>,
     bag: TileBag,
```

Comment:
I find it slightly non-intuitive that `Hands` contains the `TileBag`, and also `hands`. Could the `Hands` struct instead be named `Tiles` or something more generic? i.e. in `board.rs`:

```
hands.return_tile(letter);
```

Confused me, since that tile is going back to the bag not into a hand

Reply:
(not part of this pr though so moot pt.)

Reply:
Yeah v true. The higher level organisation could use some work. I think I put `TileBag` on `Hands` because `TileBag` was only being used by `Hands` at the time. Maybe we just remove the `TileBag` field from `Hands`.